source/demo-code/recipe-info-rough/recipePage.html

```diff
@@ -29,41 +29,106 @@
       </nav>
     <main>
         <section id="header">
+            <button style="background-color: #F1F7F7">Return</button>
 
-            <div class="recipe-favorite-button">
-                <!-- Like/Unlike recipe -->
+            <div class="title-edit-wrapper">
+                <p class = "title">Chicken Salad</p>
+                <button style="background-color: #F1F7F7">Edit</button>
             </div>
 
-            <p class = "title">
-                <!--Name of the Recipe to be populated by JS -->
-                <h1>recipeName</h1>
-            </p>
+            <div class="rating-favorite-wrapper">
+                <div class="rating">
+                    <img src="5-star.svg" class="rating" alt="autopopulated by script">
```

Comment:
could you also import the images we're going to use for 1,2,3,4 star ratings in the folder just so they're available when we need it

Reply:
Done